index.html

```diff
@@ -816,9 +821,9 @@ <h1>Speakers</h1>
 											It's great to be at FOSSASIA in Singapore! Fantastic atmosphere with lots of friendly devs, designers and business folks.
 										</div>
 										<div class="social-links vertical-align">
-											<a href="#"><i class="icon fa fa-external-link"></i></a>
+											<a href="www.bernardleong.com/biography"><i class="icon fa fa-external-link"></i></a>
```

Comment:
Add `http` before link

Reply:
@arjunpandey2003 You forgot to make change in this link. Also, please squash the commits